lib/plugins/tracestorer/index.js

```diff
@@ -19,7 +19,7 @@ module.exports = {
       case 'browsertime.chrometrace':
       case 'webpagetest.chrometrace':
         {
-          this.storageManager.writeDataForUrl(JSON.stringify(message.data, null, 0), message.name, message.url);
```

Comment:
Good catch! I'd expect Bluebird to warn about this. Maybe we didn't run it enough in non-prod mode to notice.